webapp/WEB-INF/templates/admin/plugins/workflow/modules/formspdf/modify_forms_pdf_template.html

```diff
@@ -37,6 +37,7 @@
     	<@marker_descriptions />
     </@columns>
 </@row>
+<script type="text/javascript" src='js/admin/editors/tinymce/tinymce.min.js'></script>
```

Comment:
Why not simply call <@input type=texterea richtext=true  ... > ?
(the default richtext editor might be another one)

Reply:
I've already tested this solution but it doesn't load Lutece's default text editor, so it looks like there are some back-end configurations to add. I've debugged the notifygru module but no solution yet. 
![MicrosoftTeams-image (5)](https://github.com/lutece-platform/lutece-wf-module-workflow-formstopdf/assets/125448860/390e09a7-6b96-4566-94fa-6e95425ab164)
![MicrosoftTeams-image (4)](https://github.com/lutece-platform/lutece-wf-module-workflow-formstopdf/assets/125448860/f7eefdc7-bef5-414f-af71-c71d1ea1f545)

Reply:
Adding `<@initEditor />` was the missing piece in the solution 💯